### PR TITLE
Main Form Flow

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What? 
[ROD-55](https://collaboration.homeoffice.gov.uk/jira/browse/ROD-53) - Create App Skeleton for ROD - Main Form Flow

## Testing?
Tested manually
## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

**Review by MikeMaratheftis — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+- [Node.js](https://nodejs.org/en/) - Tested against LTS
+- NPM (installed with Node.js) - Works with versions 2 and 3
+- [Redis server](http://redis.io/download) running on the default port
+
+
+### Database setup and integration
+
+If this is a first-time install get postgres running on the default port and setup a new, empty local database called `acrs`.
+Run [hof-rds-api](https://github.com/UKHomeOffice/hof-rds-api) locally for the acrs service. Hof-rds-api is an api that will read and write to your local database.
```
Comment (michaelhartHO) [inline README.md:35]: The readme still references acrs. it's worth giving it a once-over to check it's accurate for ROD

File: package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "dependencies": {
+    "accessible-autocomplete": "^2.0.4",
+    "axios": "^1.6.5",
+    "bl": "^6.0.10",
+    "busboy": "^1.6.0",
+    "bytes": "^3.1.2",
+    "dotenv": "^16.3.1",
+    "eslint-config-hof": "^1.3.1",
+    "hof": "~20.5.0",
```
Comment (michaelhartHO) [inline package.json:27]: Can you upgrade to the latest hof (^21) ?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "jquery": "^3.7.1",
+    "lodash": "^4.17.21",
+    "moment": "^2.30.1",
```
Comment (michaelhartHO) [inline package.json:30]: You may be able to remove jquery and moment.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "rod",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "Return Of Documnents (ROD)",
+  "engines": {
+    "node": ">=18.12.0"
```
Comment (michaelhartHO) [inline package.json:7]: You may be able to use a newer version of node, ^20.16

File: assets/js/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('hof/frontend/themes/gov-uk/client-js');
+const govuk = require('govuk-frontend');
+const $ = require('jquery');
+const accessibleAutocomplete = require('accessible-autocomplete');
+
+
+$('.typeahead').each(function applyTypeahead() {
```
Comment (michaelhartHO) [inline assets/js/index.js:9]: jquery can be replaced with vanilla js
`document.addEventListener('DOMContentLoaded', () => {
  document.querySelectorAll('.typeahead').forEach(function applyTypeahead(...
`
Then jquery can be removed as a dependency

**Review by michaelhartHO — APPROVED** [review]
Good foundation

File: README.md (hunk 1 above)
Comment (TemitopeAyokuHO) [reply README.md:35]: Changed

File: package.json (hunk 2 above)
Comment (TemitopeAyokuHO) [reply package.json:27]: Done

File: package.json (hunk 3 above)
Comment (TemitopeAyokuHO) [reply package.json:30]: done

File: package.json (hunk 4 above)
Comment (TemitopeAyokuHO) [reply package.json:7]: done

File: assets/js/index.js (hunk 5 above)
Comment (TemitopeAyokuHO) [reply assets/js/index.js:9]: replaced

**Pull request merged**